.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: false
+os: osx
```

Comment:
What is this solving/doing? :)

Reply:
Trying to figure out if this would make the test that fails locally also fail on CI 🤷‍♂️